src/annotator/bucket-bar.tsx

```diff
@@ -31,7 +34,11 @@ export class BucketBar implements Destroyable {
       onSelectAnnotations,
     }: BucketBarOptions
   ) {
-    this._bucketsContainer = document.createElement('div');
+    this._bucketsContainer = document.createElement('hypothesis-bucket-bar');
+    this._bucketsContainer.style.display = 'block';
+    this._bucketsContainer.style.width = '100%';
```

Comment:
Because of the way buckets are positioned, their intrinsic width is not used. Hence we need to specify a width for this container explicitly.

Reply:
> Do you think a comment here would be helpful?

Yes, will do.